.github/workflows/CI.yml

```
@@ -46,6 +46,6 @@ jobs:
         env:
           GROUP: ${{ matrix.group }}
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v4
         with:
```

Tokenless upload is not supported by codecov-action@v4. This requires the token as a Github secret and
```suggestion
        with:
          token: ${{ secrets.CODECOV_TOKEN }} # required
```